src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -546,6 +546,8 @@
                                maybe-logged)]
       (.setHandler s statistics-handler)
       (.setStopTimeout s (or shutdown-timeout default-graceful-stop-timeout))
+      (when-let [script (:post-config-script options)]
```

Comment:
Fair question.  I think my logic was that the code would be more maintainable in the long run if all of the config stuff was all consolidated in a single place in the code... so I think that my gut is that if someone was to submit a future PR that did config stuff after this function was called, we'd probably want to encourage them to move it into this function instead?  I could be persuaded, though.